### DISCUSSION TITLE
Does the broweserPermission Plugin work in headless mode?

### DISCUSSION THREAD

**Discussion by Bleyday** [kamranayub/cypress-browser-permissions#41] [Q&A] [answered]
Hello,
i am testing a web-application where allowed Notification is mandatory for its use and is checked before accesing the application. I have done the installation and the configuration for this Plugin and its working fine in headed mode while running my test. But in headless mode it keeps failing because of the not allowed notification.

Hope someone can help.

**Comment by kamranayub** [comment, marked as answer]
It should definitely work in headless mode as that is what CI is doing when running the tests. My guess is that the settings aren't taking effect for some reason. If this is still an issue, posting a reproduction in a Codesandbox or equivalent would be helpful.
